src/main/java/pl/kedziorek/mpkoperator/api/UserController.java

```diff
@@ -0,0 +1,97 @@
+package pl.kedziorek.mpkoperator.api;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import pl.kedziorek.mpkoperator.domain.Role;
+import pl.kedziorek.mpkoperator.domain.User;
+import pl.kedziorek.mpkoperator.filter.CustomAuthorizationFilter;
+import pl.kedziorek.mpkoperator.service.UserService;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.URI;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/users")
+    public ResponseEntity<List<User>> getUsers() {
```

Comment:
To bym wyrzucił na ten moment, nie będzie potrzebne zwracanie wszystkich userów w taki sposób, w późniejszym etapie przyda się zwracanie DTO usera + paginacja + filtrowanie

Reply:
Wyrzucone